### ISSUE TITLE
Version 23 on F48V4-NS Daisy Chain Issue

### ISSUE THREAD

**Issue by bpfautsch** [dpitts/f16v4_issues#40]
Daisy chaining smart receivers appears to be completely broken in version 23. Pixels attached to the first smart receiver receive all pixel data and either no or corrupt data is passed along to additional smart receivers. Running smart receivers in "dumb" mode appears to be the only functional configuration at the moment.
[Falcon_F48V4_F12A.zip](https://github.com/dpitts/f16v4_issues/files/10493900/Falcon_F48V4_F12A.zip)

**Comment by dpitts** [comment]
Hi. I used your file on two F48V4-NS controllers and all worded correctly. Maybe a wiring issue or setting issue? What receiver were you using. SmartReceiver v2.01.

**Comment by bpfautsch** [comment]
Using v2.01's. I've swapped out every piece of hardware on my LR box, power supply, cat 5, pig tails, pixels. Get green data lights on both A and B cards. Self test on the smart receivers works as should. Test on controller web interface only lights up the pixels on SR A. Been in the xlights zoom room and no one could figure out what the issue is. Daisy chaining will just not work on this controller. I can only get this board to run the smart receivers correctly in "dumb" mode. I've tried daisy chained configurations on nearly every port range on the controller as well. A friend of mine bought the same board. Had him set up same configuration, and it works on his, so there seems to be something specific to this controller. Will v23 firmware be available here soon? Maybe re-flashing the firmware will help. Done a full reset on controller multiple times with exact same result. Is there any easy way to test if the actual board is bad?
